lib/fog/openstack/requests/identity_v3/list_endpoints.rb

```diff
@@ -3,29 +3,22 @@ module Identity
     class OpenStack
       class V3
         class Real
-          def list_endpoints(options={})
-            params = Hash.new
-            params['service_id'] = options[:service_id] if options[:service_id]
```

Comment:
I wonder how to approach this in general. For some reason I would rather have if it passes all I send it, and i will have just exception back, when bad attr will be send. This way you just have to look in Openstack API documentation

If we decide to filter it, it can be done more effectively (line on one line), similar too what we do on create/update with vanilla options. Nevertheless all filtering should at least log warning with filtered out options, right? Otherwise it's hard to debug if it just becomes lost.